_posts/2018-11-29-blog-build-chrome-ringcentral-widgets-extension-for-crms.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title:  "Build Chrome RingCentral widgets extension for CRMs"
+date:   2018-11-29 13:31:07 +0800
+categories: blog
+---
+
+## Why is it need?
+
+- First, and the important reason, I guess, is Our clients need it, so for big CRM, like salesforce, we even build special app for it. For not that big CRMs, still have clients need it too, we may not build a special app for it. But we have the powerful widgets, so we can build a chrome extension easily.
+- Second, our RingCentral widgets is userful and powerful for these CRMs, we could greatly improve their work proccess.
+
+## What will we do in these extensions?
+
+In general, All the third party features of our [ringcentral-embeddable](https://github.com/ringcentral/ringcentral-embeddable) supported and some content insert:
```

Comment:
Perhaps reword this a bit.. like:
These extensions would support all the third-party interaction features of [...], and some CRM specific interactions.